commands/command_migrate_import.go

```diff
@@ -133,6 +134,10 @@ func migrateImportCommand(cmd *cobra.Command, args []string) {
 	gitfilter := lfs.NewGitFilter(cfg)
 
 	var fixups *gitattr.Tree
+	above, err := humanize.ParseBytes(migrateImportAboveFmt)
+	if err != nil {
+		ExitWithError(errors.Wrap(err, "cannot parse --above=<n>"))
```

Comment:
All the other error messages above this include the prefix `"fatal: "`, so I think it would be nice to make this do the same.

Reply:
Added.